automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookAouSpec.scala

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.dsde.workbench.leonardo.notebooks
+
+import org.broadinstitute.dsde.workbench.leonardo.{ClusterFixtureSpec, LeonardoConfig}
+import org.scalatest.DoNotDiscover
+
+/**
+ * This spec verifies expected functionality of the Terra AoU Jupyter image.
+ */
+@DoNotDiscover
+class NotebookAouSpec extends ClusterFixtureSpec with NotebookTestUtils {
```

Comment:
test needs to be added https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala#L93